flow/platforms/sky130hd_fakestack/pdn.tcl

```diff
@@ -0,0 +1,39 @@
+####################################
+# global connections
+####################################
+add_global_connection -defer_connection -net {VDD} -inst_pattern {.*} -pin_pattern {^VDD$} -power
+add_global_connection -defer_connection -net {VDD} -inst_pattern {.*} -pin_pattern {^VDDPE$}
+add_global_connection -defer_connection -net {VDD} -inst_pattern {.*} -pin_pattern {^VDDCE$}
+add_global_connection -defer_connection -net {VDD} -inst_pattern {.*} -pin_pattern {VPWR}
+add_global_connection -defer_connection -net {VDD} -inst_pattern {.*} -pin_pattern {VPB}
+add_global_connection -defer_connection -net {VSS} -inst_pattern {.*} -pin_pattern {^VSS$} -ground
+add_global_connection -defer_connection -net {VSS} -inst_pattern {.*} -pin_pattern {^VSSE$}
+add_global_connection -defer_connection -net {VSS} -inst_pattern {.*} -pin_pattern {VGND}
+add_global_connection -defer_connection -net {VSS} -inst_pattern {.*} -pin_pattern {VNB}
+global_connect
+####################################
+# voltage domains
+####################################
+set_voltage_domain -name {CORE} -power {VDD} -ground {VSS}
+####################################
+# standard cell grid
+####################################
+define_pdn_grid -name {grid} -voltage_domains {CORE}
+add_pdn_stripe -grid {grid} -layer {met1} -width {0.48} -pitch {5.44} -offset {0} -followpins
+add_pdn_stripe -grid {grid} -layer {met4} -width {1.600} -pitch {27.140} -offset {13.570}
+add_pdn_stripe -grid {grid} -layer {met5} -width {1.600} -pitch {27.200} -offset {13.600}
+add_pdn_connect -grid {grid} -layers {met1 met4}
+add_pdn_connect -grid {grid} -layers {met4 met5}
```

Comment:
Does it make sense to have the power grid on 4&5 in your 9 layer stack?

Reply:
So you want to handle this in another PR?

Reply:
Yes  I will file a JIRA -